prov/sockets/src/sock_ep.c

```diff
@@ -673,8 +674,9 @@ static int sock_ep_close(struct fid *fid)
 			SOCK_LOG_DBG("Failed to signal\n");
 
 		if (sock_ep->attr->cm.listener_thread &&
-		    pthread_join(sock_ep->attr->cm.listener_thread, NULL)) {
-			SOCK_LOG_ERROR("pthread join failed (%d)\n", errno);
+			pthread_join(sock_ep->attr->cm.listener_thread, NULL)) {
+			SOCK_LOG_ERROR("pthread join failed (%d)\n",
+				       ofi_sockerr());
```

Comment:
I wouldn't use sockerr for pthread calls.

Reply:
replaced by `ofi_syserr()`